packages/terra-data-grid/src/DataGrid.module.scss

```diff
@@ -9,19 +9,7 @@
     width: 100%;
```

Comment:
I would not expect the data-grid-container element to have an overflow.  The Terra Table already had an element with an overflow.